src/charactersheet/viewmodels/dm/status_line/index.js

```diff
@@ -41,65 +40,65 @@ export function PartyStatusLineViewModel(params) {
     };
 
     self.getStatusLine = function(players) {
-        var chat = ChatServiceManager.sharedService();
-        if (chat.currentPartyNode == null || players.length < 1) { return ''; }
-        var totalHealthiness = 0;
-        var totalMagic = 0;
-        var totalTrackedAbilities = 0;
-        var healthinessModified = false;
-        var magicModified = false;
-        var trackedAbilitiesModified = false;
-        var statuses = [];
-        var numberOfHealthinessStatuses = 0;
-        var numberOfMagicStatuses = 0;
-        var numberOfTrackedStatuses = 0;
-
-        for (const player of players) {
-            if (player.playerType() === PlayerTypes.character.key) {
-                if (player.healthinessStatus()) {
-                    totalHealthiness += player.healthinessStatus().value();
-                    healthinessModified = true;
-                    numberOfHealthinessStatuses++;
-                }
-                if (player.magicStatus()) {
-                    totalMagic += player.magicStatus().value();
-                    magicModified = true;
-                    numberOfMagicStatuses++;
-                }
-                if (player.trackedStatus()) {
-                    totalTrackedAbilities += player.trackedStatus().value();
-                    trackedAbilitiesModified = true;
-                    numberOfTrackedStatuses++;
-                }
-            }
-        }
-
-        if (healthinessModified) {
-            statuses.push(StatusWeightPair.determinePhraseAndColor(getHealthTypeEnum(), totalHealthiness / numberOfHealthinessStatuses));
-        }
-
-        if (magicModified) {
-            statuses.push(StatusWeightPair.determinePhraseAndColor(getMagicTypeEnum(), totalMagic / numberOfMagicStatuses));
-        }
-
-        if (trackedAbilitiesModified) {
-            statuses.push(StatusWeightPair.determinePhraseAndColor(getTrackedTypeEnum(), totalTrackedAbilities / numberOfTrackedStatuses));
-        }
-
-        if (statuses.length < 1) { return ''; }
-
-        return 'Your party is ' + statuses.map(function(e, i, _) {
-            var status = '<span class="text-' + e.color + '">' + e.status + '</span>';
-            if (statuses.length > 1 && i == statuses.length - 1) {
-                return 'and ' + status;
-            } else if (statuses.length > 2) {
-                return status + ',&nbsp;';
-            } else if (statuses.length > 1) {
-                return status + '&nbsp;';
-            } else {
-                return status;
-            }
-        }).join('') + '.';
+//         var chat = ChatServiceManager.sharedService();

```

Comment:
any plans for this?

Reply:
Yeah I forgot to implement it here, but I've added it in my other PR.